api/provider/aws/deploy_read.go

```diff
@@ -0,0 +1,64 @@
+package aws
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func (d *DeployProvider) Read(deployID string) (*models.Deploy, error) {
+	fqDeployID := addLayer0Prefix(d.Config.Instance(), deployID)
```

Comment:
This implementation will need to change w the new arn stuff - just marking it for now since I know it's WIP